src/classes/REL_Relationships_TEST.cls

```diff
@@ -359,22 +378,56 @@ public class REL_Relationships_TEST {
     public static void testAutoCreateContactUpdate(){
         setupRelationshipTestData(null);
 
-        //reset the process controller to mimic a second user action
-        //Relationships_ProcessControl.hasRun = false;
+        //insert a new Contact
+        Contact c = new Contact(LastName='ReportingIn', ReportsToId = c1.Id);
```

Comment:
I think I am noticing a pattern here. Perhaps whomever originally created this had their own style of string assignment no space, other assignments get spaces?